Dockerfile-testing

```diff
@@ -0,0 +1,34 @@
+# Use the official Python base image
+FROM python:3.11-bookworm
+
+# Set the working directory in the container
+WORKDIR /app
+
+# Copy the requirements file to the container
+COPY requirements.txt .
```

Comment:
Move this further down, any changes to requirements will cause the apt execution to restart

Reply:
#95